charts/generic-govuk-app/templates/deployment.yaml

```diff
@@ -127,9 +136,9 @@ spec:
           {{- end }}
           securityContext:
             allowPrivilegeEscalation: {{ .Values.securityContext.allowPrivilegeEscalation }}
-            readOnlyRootFilesystem: true
+            readOnlyRootFilesystem: {{ .Values.securityContext.readOnlyRootFileSystem }}
             capabilities:
-              drop: ["ALL"]
+              drop: {{ .Values.securityContext.capabilities.drop }}
```

Comment:
Why do we need to read these in as Values?

Reply:
No particular reason. Mostly just being consistent with extracting everything out to values.

Reply:
If we're not changing it (and we wanna actually enforce this value anyway) - there's no need to have it as a variable. Just adds complexity to the configuration.

Reply:
Yep that's fair. I've pushed up now!